src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/PartialUpdateSpec.groovy

```diff
@@ -85,10 +85,6 @@ class PartialUpdateSpec extends HealthCheckSpecification {
                         field   : "maxLatency",
                         newValue: 12345
                 ],
-                [
-                        field   : "maxLatencyTier2",
-                        newValue: 23456
-                ],
```

Comment:
As you removed a positive test case - should we add  new one? When a flow has maxLatency and you are trying to PATCH maxLatencyTier2

Reply:
test was failing due to new validation
i review other, and wanted to implement it with another fix
but you are right - so returning it back with right precondition to allow test run properly